include/extractor/traffic_lights.hpp

```diff
@@ -19,6 +20,36 @@ enum Direction
 };
 } // namespace TrafficLightClass
 
+// Stop Signs tagged on nodes can be present or not. In addition Stop Signs have
+// an optional way direction they apply to. If the direction is unknown from the
+// data we have to compute by checking the distance to the next intersection.
```

Comment:
Hmm, looks like identifying the direction will require something additional to the traffic signal logic.

Reply:
> If the direction is unknown from the data we have to compute by checking the distance to the next intersection.

Do you mean this sentence? Tbh it was a copy-paste from some of older PRs and I noticed it only now...

Reply:
Yep.

Reply:
It looks like ~77% of stop signs tagged with direction.

I will take a look what it will take to compute this "distance to the nearest intersection" - should be just a couple of heuristics, but not sure how to do that from code point of view, but if it is TOO difficult to be part of this PR I would probably propose to just ignore stop signs without direction for the time being and then return to it in separate PRs.

https://taginfo.openstreetmap.org/tags/highway=stop#combinations
<img width="1264" alt="Screenshot 2022-10-28 at 16 39 59" src="https://user-images.githubusercontent.com/266271/198652260-98998f07-a68b-4558-a787-1c429c703643.png">